perllib/FixMyStreet/Cobrand/UK.pm

```diff
@@ -481,10 +481,35 @@ sub _updates_disallowed_check {
         return $cfg unless !$body_comment_user;
     } elsif ($cfg eq 'notopen311-open') {
         return $cfg unless !$body_comment_user && $open;
+    } elsif ($cfg eq 'open-or-reporter') {
```

Comment:
Probably worth adding this new value to an `ELSIF` in `_updates_disallowed_hint.html`, and seeing if it fits into "Can I make updates to my report?" in `fixmystreet-uk-councils/about/faq-en-gb.html`.

Reply:
Oops, no it's not. Sorry!